org.salutatoi.Cagou.json

```diff
@@ -0,0 +1,1463 @@
+{
+    "app-id": "org.salutatoi.Cagou",
+    "default-branch": "0.7.0",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "sat_wrapper",
+    "finish-args": [
+        "--socket=session-bus",
+        "--socket=x11",
+        "--share=ipc",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
```

Comment:
Python 2 is available in https://github.com/flathub/shared-modules/

Reply:
I have seen indeed, but after doing that one unfortunately. I'll update to use the official one.